src/frameserver/PointGreyCam.cpp

```diff
@@ -539,7 +541,7 @@ void PointGreyCam<T>::setupAsyncTrigger(int trigger_mode,
         trigger_mode != 1  ||
         trigger_mode != 13 ||
         trigger_mode != 14) {
-        throw (rte("Trigger mode is unsupported."));
+      //throw (rte("Trigger mode is unsupported."));
```

Comment:
The free-running block was there, just a few blocks down. Line 542 still throws an error, even though line 531 outputs correct mode. Could it be a conflict between variable types, eg. int vs float passed from bash? I don't know where to start debugging this